src/platforms/rcore_android.c

```diff
@@ -74,6 +74,7 @@ typedef struct {
 // Global Variables Definition
 //----------------------------------------------------------------------------------
 extern CoreData CORE;                   // Global CORE state context
+#define CORE_WINDOW CORE.Window
```

Comment:
I think defining the macro again in each rcore_*.c is not necessary, just rcore.c should be enough, since it includes those files.